aiida_common_workflows/workflows/dissociation.py

```diff
@@ -0,0 +1,200 @@
+# -*- coding: utf-8 -*-
+"""
+Workflow calculating the dissociation curve of diatomic molecules.
+It can use any code plugin implementing the common relax workflow.
+"""
+import inspect
+from aiida import orm
+from aiida.common import exceptions
+from aiida.engine import WorkChain, append_, calcfunction
+from aiida.plugins import WorkflowFactory
+from aiida_common_workflows.workflows.relax.generator import RelaxType, SpinType
+from aiida_common_workflows.workflows.relax.workchain import CommonRelaxWorkChain
+
+
+def validate_inputs(value, _):
+    """Validate the entire input namespace."""
+    if 'distances' not in value:
+        if any(key not in value for key in ['distances_count', 'distance_min', 'distance_max']):
+            return 'neither `distances` nor the `distances_count`, `distance_min`, and `distance_max` set were defined.'
+    if 'distance_min' in value:
+        if value['distance_min'] >= value['distance_max']:
```

Comment:
what if `distance_max` is not defined here?

Reply:
You are right. It fails when `distance_min` and `distances` are defined, but not `distance_max`. a bit of a weird case but I'll change it in next commit

Reply:
The change is all good, I am just wondering now if it isn't a bit weird to allow an empty list for `distances`. Anyway, I will merge this now and then we can address this later

Reply:
It is not allowed! There is the `validator_distances` that takes care of it later on!